tests/data/test.htm

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"

```

Comment:
Make sure this and all `__pycache__` folders are in the `.gitignore`. You don't wanna see these on github.

Reply:
Checked: `__pycache__` folders are in the `.gitignore` and do not appear on github on this branch.